k9mail/src/main/res/values-bg/strings.xml

```diff
@@ -788,7 +788,7 @@ K-9 Mail е мощен, безплатен имейл клиент за Андр
   <string name="settings_exporting">Настройки на изнсянето...</string>
   <string name="settings_importing">Настройки на внасянето...</string>
   <string name="settings_import_scanning_file">Сканиране на файл...</string>
-  <string name="settings_export_success">Изнесенети настройки са запазени като <xliff:g id="filename">%s</xliff:g></string>
```

Comment:
Normally, I like refactorings that improve names/identifiers. In this case I'd prefer this to stay the same. Otherwise all translations of this string will be invalidated on Transifex.